fe/fe-core/src/main/java/org/apache/doris/analysis/PartitionNames.java

```diff
@@ -111,6 +111,10 @@ public long getCount() {
 
     @Override
     public void analyze(Analyzer analyzer) throws AnalysisException {
+        analyze();
+    }
+
+    public void analyze() throws AnalysisException {
```

Comment:
use PartitionNamesInfo, PartitionNames don't need to be changed

Reply:
done - please review